Classes/Hooks/HookProvider.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Hooks;
+
+use OliverKlee\Seminars\Hooks\Interfaces\Hook;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Provides functions for unified hooks
```

Comment:
Nitpick: Please generally add the full stop at the end of class or method descriptions that would be a complete sentence if "This class/method …" were added at the start.